server/app/config.js

```diff
@@ -105,31 +105,30 @@ app.use("/api", apiRouter);
 // 2. Ensure that the `reactBuildPath` points to the correct directory where your client's build artifacts are located.
 
 const reactBuildPath = path.join(__dirname, "/../../client/dist");
-const publicFolderPath = path.join(
-  __dirname,
-  "/../public/assets/images/upload"
-);
+const publicFolderPath = path.join(__dirname, "/../public");
 
 // Serve react resources
 
 app.use(express.static(reactBuildPath));
 
 // Serve server resources
 
-app.get("*.*", express.static(publicFolderPath, { maxAge: "1y" }));
+app.use(
+  "/assets",
+  express.static(path.join(publicFolderPath, "assets"), { maxAge: "1y" })
+);
 
```

Comment:
same , hopefully the features based on it will not be affected

Reply:
It's good 👍